tiapp.xml

```diff
@@ -135,4 +131,5 @@
         <module platform="iphone">facebook</module>
     </modules>
     <sdk-version>7.1.0.GA</sdk-version>
+    <property name="appc-app-id" type="string">5ae1a62f6d9244226ec9bed5</property>
```

Comment:
I think you commited your local appc-registered app here. Please restore the tiapp.xml